pkg/pillar/cmd/domainmgr/metric.go

```diff
@@ -70,8 +71,31 @@ func getAndPublishMetrics(ctx *domainContext, hyper hypervisor.Hypervisor) {
 			// We clear the memory so it doesn't accidentally get
 			// reported.  We keep the CPUTotal and AvailableMemory
 			dm.UsedMemory = 0
+			dm.MaxUsedMemory = 0
 			dm.UsedMemoryPercent = 0
 		}
+
+		config := lookupDomainConfig(ctx, status.Key())
```

Comment:
Yeah you would panic here snd not below. But only with xen where we get the niluuid one for dom0. Explains ehh ring0 on xen fails I think